addons/mail/static/src/models/thread/thread.js

```diff
@@ -810,6 +810,16 @@ function factory(dependencies) {
             return this.pendingFoldState || this.serverFoldState;
         }
 
+        /**
+         * @private
+         */
+        _computeHasSeenIndicators() {
+            if(this.mass_mailing) {
+                return false;
+            }
+            return this.members.length === 2;
```

Comment:
This should be based on channel type instead of number of members (because this might be unreliable, especially for livechat, but it is also theoretically possible for `channel_get` to return a "chat" with multiple members for which we probably want to keep the feature).